app/assets/javascripts/components/utils.js.coffee

```diff
@@ -132,3 +132,9 @@ class @Utils
       .fail (xhr, status, error) ->
         alert('Error: ' + error)
         callback()
+
+  @select2TagClickHandler: (e, elem) ->
+    if e.which == 1
```

Comment:
It feels like we should always open in a new window since otherwise you could lose editing you've done?